internal/controller/tortoise_controller_test.go

```diff
@@ -477,6 +487,14 @@ var _ = Describe("Test TortoiseController", func() {
 			runTest(filepath.Join("testdata", "mutable-autoscalingpolicy-remove-horizontal-2"))
 		})
 	})
+	Context("automatic switch to emergency mode", func() {
+		It("HPA scalingactive condition false", func() {
+			runTest(filepath.Join("testdata", "reconcile-automatic-emergency-mode-hpa-condition"))
+		})
+		It("HPA scalingactive no metrics", func() {
+			runTest(filepath.Join("testdata", "reconcile-automatic-emergency-mode-hpa-no-metrics"))
+		})
```

Comment:
Can you add one more test case which tests that tortoise gets back to `tortoisePhase: Working` when HPA is back to normal?